src/widgets/index.js

```diff
@@ -21,6 +21,7 @@ export { TextEditor } from './TextEditor';
 export { TextInput } from './TextInput';
 export { ToggleBar } from './ToggleBar';
 export { ToggleSelector } from './ToggleSelector';
+export { NewExpiryDateInput } from './NewExpiryDateInput';
```

Comment:
Trying to keep this out of the data table?

Reply:
The previous one wasn't and the use case seems pretty (word)

Reply:
Pretty... specific to mobile?